src/Application/AnalyzeCommand.php

```diff
@@ -444,4 +464,47 @@ private function getFinder(array $paths): array
 
         return iterator_to_array($found);
     }
+
+    private function generateBaseline(TwigStanAnalysisResult $result, OutputInterface $output, bool | string $generateBaselineFile): void
+    {
+        if ($generateBaselineFile === false) {
+            return;
+        }
+
+        if ($generateBaselineFile === true) {
+            $generateBaselineFile = 'twigstan-baseline.neon';
+        }
+
+        $extension = pathinfo($generateBaselineFile, PATHINFO_EXTENSION);
```

Comment:
Can we use `\Symfony\Component\Filesystem\Path` for this?

Reply:
yes, of course. done